Wrote an assignment jv-logger.

ASelivonchyk opened the pull request:
- Configured logger that write logs to file and console;
- Completed all TODO statements.

ASelivonchyk committed:
Wrote an assignment jv-logger.
- Configured logger that write logs to file and console;
- Completed all TODO statements.

MaliukDaria commented on src/main/java/mate/academy/service/OrderServiceImpl.java:
```diff
@@ -4,27 +4,27 @@
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderService.class);
```
```suggestion
    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
```

ASelivonchyk committed:
Fixed bugs.
- changed OrderService.class to OrderServiceImpl.class in OrderServiceImpl class field.

MaliukDaria reviewed (approved):
🚀
